docs/en/advanced_guides/supported_tasks/lidar_det3d.md

```diff
@@ -28,14 +28,14 @@ mmdetection3d
 │   ├── kitti
 │   │   ├── ImageSets
 │   │   ├── testing
-│   │   │   ├── calib
```

Comment:
Why remove calib

Reply:
Seem all the transform matrix has been moved to the corresponding JSON file

Reply:
But we still need the raw data to generate the info file?